Configurations/windows-makefile.tmpl

```diff
@@ -220,6 +220,7 @@ test: tests
 	set BLDTOP=$(BLDDIR)
 	set RESULT_D=$(BLDDIR)\test\test-runs
 	set PERL=$(PERL)
+	set OPENSSL_ENGINES=$(BLDDIR)\engines
```

Comment:
Have you tested that this works? I think that it needs to be an absolute directory...

Reply:
oops....